meta-intel-edison-bsp/recipes-kernel/linux/files/0001-phy-ti-tusb1210-write-to-scratch-on-power-on.patch

```diff
@@ -0,0 +1,25 @@
+From 9826a1ce331e0418aadf47c33be421452ed9e904 Mon Sep 17 00:00:00 2001
+From: Ferry Toth <ftoth@exalondelft.nl>
+Date: Sun, 26 Feb 2023 22:07:44 +0100
+Subject: [PATCH 1/1] phy: ti: tusb1210: write to scratch on power on
```

Comment:
Have you tried to discuss / upstream this one?

Reply:
I believe I did but let me look it up.

Reply:
So, it means that the problem itself is unknown (root cause)...

Reply:
Yes, it's harmless but at the same time should have no effect at all. Maybe just post it to ML and to you and Hans and see where it goes?